packages/core/src/chains/cosmos.ts

```diff
@@ -0,0 +1,100 @@
+// To re-generate these, clone the cosmos/chain-registry repo and run this jq script:
```

Comment:
For translating, I think using chainID prefixes or wildcards would make sense -- e.g. akashnet-2 will become akashnet-3 at some point in the future, and we can assume the address prefix will remain the same

Reply:
OK... every `chain.json` has both `chain_name` (`akash`) and `chain_id` (`akashnet-2`) - should we use the `chain_name` as the `ChainId` instead of `chain_id`?

Reply:
Well, a `chain_id` and `blockhash` uniquely identifies a block, while a `chain_name` and `blockhash` doesn't technically do the same thing (in practice block hashes are unique but block explorers list them by chain ID: https://www.mintscan.io/cosmos/blocks/13277881)